crates/codegen/language/definition/src/model/manifest.rs

```diff
@@ -144,3 +189,36 @@ pub enum BuiltInLabel {
     LeadingTrivia,
     TrailingTrivia,
 }
+
+#[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
```

Comment:
nit suggestion: moving the added `BuiltIn` types to a new module `src/model/built_ins.rs`